### ISSUE TITLE
Use vue-toastify from Vuex Store

### ISSUE THREAD

**Issue by bhaskar-nair2** [nandi95/vue-toastify#14]
Hey, I was looking for a way to generate messages from my Vuex store.
I have tried 
```js
Vue.$vToastify.error('easy-peasy');
this.$vToastify.error('easy-peasy');
```
both fail as `vToastify` is undefined in both..

Please note, I am using a dynamic store, which gets registered at runtime.

Thanks!

**Comment by nandi95** [comment]
@bhaskar-nair2 
This can be looked at, at some point in the future.
Currently however, the priority is creating a decent documentation site.
If you you're willing I accept PRs in the meantime.

**Comment by bhaskar-nair2** [comment]
I understand.. Lemme see if I can do it..

**Comment by nandi95** [comment]
@bhaskar-nair2 It seems you should ba able to access it like so:
`this._vm.$vToastify.success("called from store");`
OR
`Vue.prototype.$vToastify.success("called from store");`

**Comment by nandi95** [comment]
Addressed in 1.8.0
You may now use it in vuex like:
```javascript
import Vue from "vue";

const mutations = {
  updateValue(state, value) {
    state.value = value;
    Vue.$vToastify("Successfully updated value!");
    // OR
    this._vm.$vToastify("Successfully updated value!")
  }
};
```
